src/main/java/com/notably/logic/commands/suggestion/SearchSuggestionCommand.java

```diff
@@ -78,8 +79,12 @@ public void execute(Model model) {
                             int frequency = blockBodies.length - 1;
                             String displayText = absolutePath.getStringRepresentation();
                             Runnable action = () -> {
-                                OpenCommand openCommand = new OpenCommand(absolutePath);
-                                openCommand.execute(model);
+                                try {
+                                    OpenCommand openCommand = new OpenCommand(absolutePath);
+                                    openCommand.execute(model);
+                                } catch (CommandException ex) {
+                                    throw new RuntimeException(ex.getMessage());
```

Comment:
Actually, should we throw an `AssertionError` here, just to be specific? Also, let's add an inline comment here to explain why there shouldn't be any error while executing the `openCommand`? @johannagwan Please advise!

Reply:
thanks @kevinputera for pointing this out! 

Regarding why there shouldn't be any error while executing the `OpenCommand`: When the user searches based on a keyword, the notes suggested will definitely be able to be opened as the block actually exists, so `NoSuchBlockException` will not be thrown. Thus, the `AssertionError` (or `RuntimeException`) should never be called. So yehhh it's good if we can add the rationale here.

Thanks for the sik werk @ljiazh3ng !

Reply:
Roger, will change it to Assertion Error